docs/migrate-to-ory/auth0.mdx

```diff
@@ -0,0 +1,140 @@
+---
+id: auth0
+title: Guide to migrate from Auth0 to Ory
+sidebar_label: Auth0
+slug: migrating-from-auth0
+---
+
+## Migrating from Auth0 to Ory
+
+This guide helps you to migrate from Auth0 to Ory. The base example assumes that your Auth0 setup includes a database connection
+and email and password for credentials. For other setups this guide serves as a good starting point, which you can fit to your
+usecase.
+
+In this guide, you'll:
+
+1. open a support ticket to get your hashed passwords from Auth0.
+
+2. run a bulk user export to get your complete user data.
+
+3. create identities in Ory Network.
+
+### Export password hashes
+
+Because password hashes are considered sensitive information, Auth0 doesn't export them as part of the general export process. You
+must file a support ticket to get the hashes and other password related information.
+
+! **When you are exporting hashed passwords, neither you, nor Auth0 or the Ory system has access to the plain text password** !
+
+When you have exported the hashed passwords from Auth0 and imported them into Ory Kratos, your users will be able to log in with
+the same password they used previously.
+
+**Tip:**
+
+> If you aren't able to export the hashes, due to the plan you are on or the migration timelines, one alternative is to have every
+> user change their password using the forgot password flow. You can still import all the user data.
+
+To start the process of obtaining the hashes, open a support ticket. From your dashboard, navigate to Get Support, and then open a
+ticket by scrolling to the bottom of the support page.
```

Comment:
Use the right formatting when talking about UI elements. Refer to the guidelines for more info.